avocado_vt/test.py

```
@@ -363,3 +374,170 @@ def _runTest(self):
+
+        :raises: SelectorError: If no available nodes match the specified selectors.
+        """
+        self._cluster_partition = cluster.create_partition()
+        for node in self.params.objects("nodes"):
+            node_params = self.params.object_params(node)
+            node_selectors = node_params.get("node_selectors")
+            _node = selector.select_node(cluster.idle_nodes, node_selectors)
+            if not _node:
+                raise selector.SelectorError(
+                    f'No available nodes for "{node}" with "{node_selectors}"'
+                )
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)